modules/ROOT/pages/_partials/api-experience-hub-usage.adoc

```diff
@@ -0,0 +1,20 @@
+= API Experience Hub Usage Reports
+
+include::partial$api-eh-metrics.adoc[tag=intro]
+
+== Access API Experience Hub Usage Reports
+
+include::partial$usage-reports-tasks.adoc[tag=accessUsageReportsNew]
+
+image::api-eh-usage-report.png[Usage Reports interface showing an API Experience Hub usage report that contains a card and table of usage data]
```

Comment:
y'wanna put that alt text in quotes, just for grins?

Reply:
@hannanelson The reuse in this doc is highly convoluted. Turns out the stuff in this file being used.